source/guide_invoiceninja.rst

```diff
@@ -157,22 +164,27 @@ After that you can :manual_anchor:`add a new supervisord service <daemons-superv
 
 ::
 
- /usr/bin/php /var/www/virtual/$USER/invoiceninja/artisan queue:work --daemon
+ /usr/bin/php /var/www/virtual/$USER/artisan queue:work --daemon
```

Comment:
```suggestion
 /usr/bin/php /var/www/virtual/$USER/html/artisan queue:work --daemon
```
based on the cronjob fix, this should read as above?

Reply:
absolutely yes